package.json

```diff
@@ -55,6 +58,7 @@
     "grunt-template-jasmine-requirejs": "0.2.3",
     "imports-loader": "0.6.5",
     "json-loader": "0.5.4",
+    "junit-report-builder": "1.1.1",
```

Comment:
Do we need shrinkwrap for the sign-in-widget? 
We have control over when it merges master and deploys, no?